plonetheme/onegovbear/theme/scss/grid.scss

```diff
@@ -1,71 +0,0 @@
-$column-width: 56px;
-$gutter-width: 18px;
-$margin-width: 18px;
-$columns: 16;
-
-$gridsystem-width: $columns * $column-width + ($columns - 1) * $gutter-width + 2 * $margin-width;
-
-@for $x from 1 through $columns {
```

Comment:
@maethu are there components still using the grid system or can we just get rid of it?

Anyway:
I the `index.html` there are still some `position-X` and `width-Y` classes which can be removed, if I'm getting it right, because it explicitly styled by id or another class by using the `cell` / `position` / `width` mixins, right?

If we get rid of all those classes in the `index.html` we could probably keep the `grid.scs` (since it does not apply to our theme skeleton any more) and maybe not break addons still relying on it?
Just an idea :wink: